ooxml/XSSF/UserModel/XSSFVMLDrawing.cs

```diff
@@ -116,7 +117,7 @@ internal void Read(Stream is1)
             //Stream vmlsm = new EvilUnclosedBRFixingInputStream(is1); --TODO:: add later
             
              doc.LoadXml(
-                  data.Replace("<br>","").Replace("</br>", "")
+                  data.Replace("<br>", "<br/>").Replace("</br>", "<br/>")
```

Comment:
Don't change this code. I remember this is used to fix something.

Reply:
There were issues with the previous fix. This fix has added the test case  `TestEvilUnclosedBRFixing`, the previous fix make it failed.  I think it was used to fix the Unclosed BR tag issue.